app/services/publish_teacher_training/subject/import.rb

```diff
@@ -33,36 +37,52 @@ def fetch_subject
           subject_ids: secondary_subject_ids,
           subjects_data:,
         )
+
+        # Sync Modern Language Subjects
+        sync_subjects(
+          subject_area: :secondary,
+          subject_ids: modern_language_subject_ids,
+          subjects_data:,
+          parent_subject: ::Subject.find_by(name: "Modern Languages"),
```

Comment:
Would this be safer as:

```suggestion
          parent_subject: ::Subject.find_by!(name: "Modern Languages"),
```

so it explodes if the MFL subject can't be found?

Otherwise I think this will run with `parent_subject: nil`, so the MFL subjects become top level.